aten/src/ATen/native/cpu/group_norm_kernel.cpp

```diff
@@ -1367,7 +1542,7 @@ void GroupNormBackwardKernelImpl(
   // It will keep module parameters in acc dtype i.e. float
   // while input/output will be in BFloat16.
   // Using parameters in BFloat16 will cause high precision loss.
-  const bool mixed_type = is_mixed_type(dY, gamma);
+  const bool mixed_type = is_mixed_type(dY, mean);
```

Comment:
The comment doesn't correlate well with the code any more? Can you revise the comment?

Reply:
Sure, revised at https://github.com/pytorch/pytorch/pull/100234.